app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -910,7 +909,7 @@ def rbac_group_get_details(id)
     [@group.get_managed_filters].flatten.each do |f|
       @filters[f.split("/")[-2] + "-" + f.split("/")[-1]] = f
     end
-    rbac_build_myco_tree                              # Build the MyCompanyTags tree for this user
+    @tags_tree = TreeBuilderTags.new(:tags, :tags_tree, @sb, true, @edit, @filters, @group)
```

Comment:
In recent refactoring PRs the 5th and further args are passed together as a hash with descriptive keys.

That seems more readable to me